### PR TITLE
fix event bug due to api parity

### LINKED ISSUE #1040: Design: Event notification naming scheme / layout
Suggestions were made re: event notifications in the realm of:

- {Noun}:{Verb}

- {Linode Label}:{Event Action}

Regarding time stamp, up for discussion. Please post comments if other ideas.

### LINKED ISSUE #1086: Events API parity

### PR DESCRIPTION
**Pull request by abemassry**
closes #1086

### CODE REVIEW

File: src/components/Notifications.js [hunk 1]
```diff
@@ -132,9 +131,7 @@ export function Notification(props) {
 
 Notification.propTypes = {
   id: PropTypes.number.isRequired,
-  label: PropTypes.string.isRequired,
```
Comment (mparke) [inline src/components/Notifications.js]: updated elsewhere but `entity` dropped from `propTypes`?

Comment (abemassry) [inline src/components/Notifications.js]: i tried adding entity into propTypes but I got an error.  When I took it away I got no errors and the pages behaved correctly.  Tests were also passing.  I thought it was weird, I can go over it with you.

Comment (mparke) [inline src/components/Notifications.js]: okay, is fine for now, I'll look if it can be added when updating the event naming

Comment (eatonphil) [inline src/components/Notifications.js]: What error did you get @abemassry?

Comment (mparke) [inline src/components/Notifications.js]: @eatonphil going to pick up where @abemassry left off, can address this

@abemassry can you update this to not close the api issue

**Pull request merged**